app/controllers/claims/schools/claims_controller.rb

```diff
@@ -33,6 +33,15 @@ def update
     end
   end
 
+  def confirm; end
+
+  def submit
+    submit_form = Claim::SubmitForm.new(claim: @claim)
+    submit_form.save!
+
+    redirect_to confirm_claims_school_claim_path(@school, @claim)
+  end
```

Comment:
This isn't a part of a HTML form. Should we make this a service instead? `Claim::Submit.call(claim: @claim)`

Reply:
Sorry, I think I wasn't too clear.

```suggestion
  def submit
    Claim::Submit.call(claim: @claim)

    redirect_to confirm_claims_school_claim_path(@school, @claim)
  end
```

I think the redirection and controller logic should stay put, but the stuff within the `SubmitForm` doesn't handle a form submission, there isn't a form at all actually, it marks a draft as submitted. That logic looks to belong within a service.